ui/src/shared/apis/v2/query.ts

```diff
@@ -19,8 +21,21 @@ interface XHRError extends Error {
 export const executeQuery = async (
   url: string,
   query: string,
-  language: InfluxLanguage = InfluxLanguage.Flux
+  language: InfluxLanguage = InfluxLanguage.Flux,
+  variables?: {[key: string]: string}
 ): Promise<ExecuteFluxQueryResult> => {
+  let preamble = ''
+
+  if (variables && language === InfluxLanguage.Flux) {
+    preamble = _.reduce(
+      variables,
+      (result, value, name) => `${result}${name} = ${value}\n`,
+      ''
+    )
+  }
+
+  query = `${preamble}${query}`
```

Comment:
```suggestion
  query = `${preamble}\n${query}`
```
I am a huuuge fan of line breaks everywhere all of the time :)

Reply:
I left it without because when there is a preamble the preamble has a line break and when there is no preamble then the query is unchanged.